.packit.yaml

```
@@ -14,7 +14,15 @@
 
 # https://packit.dev/docs/configuration/
 
-specfile_path: packages/fedora/openarm-can.spec
+specfile_path: openarm-can.spec
```

Could you use `packages/fedora/` not the top-directory?


```suggestion
specfile_path: packages/fedora/openarm-can.spec
```